.eslintrc

```diff
@@ -29,5 +29,19 @@
         "every": [ "id" ]
       }
     }]
-  }
+  },
+  "overrides": [
+    {
+      "files": ["html/**/*.js"],
+      "rules": {
+        "max-len": "off"
+      }
+    },
```

Comment:
Thanks for the PR, @motionsuggests ! We don't want to disable max-len for all the files. We take it on more of an exception basis than as an overall rule. Would you mind removing this and updating with staging? Everything else looks good!

Reply:
Sorry for not getting to this sooner. Should I just go through and disable max-len inline where necessary? Like this
```js
export default {
  title: 'Components/Input/Checkbox',
  decorators: [(story) => `<div class="sprk-o-Box">${story()}</div>`],
  /* eslint-disable max-len */
  parameters: {
    info: `
${markdownDocumentationLinkBuilder('input')}
- Elements containing role="checkbox" must also include the
 aria-checked attribute to expose the checkbox's state to assistive technology.
- Since a checkbox is an interactive control, it must be focusable
and keyboard accessible. If the role is applied to a non-focusable element,
use the tabindex attribute to change this.
- Change the value of the aria-checked attribute
dynamically when the checkbox is activated.

###### Ensure all related checkboxes are grouped together using one of the following methods:

- Fieldset and legend tags
- ARIA groups role="group" and aria-label or aria-labelledby
- aria-labelledby that points to the same
 element for every checkbox in the group
    `,
    docs: { iframeHeight: 180 },
  },
};
/* eslint-enable max-len */
```

Reply:
@guern1kn This is ready for review. I went ahead and updated everywhere to use 
```
/* eslint-disable max-len */
/* eslint-enable max-len */
```

Reply:
Hi @motionsuggests thank for all you work on this so far.

So for places like 
```
###### Ensure all related checkboxes are grouped together using one of the following methods:
```

would turn into:
```
###### Ensure all related checkboxes are grouped together
using one of the following methods:
```

We typically break the line into the next to avoid excessive line length. The Gatsby parses it fine as long as it's immediately on the next line.

--------

**I'm also seeing a lot of places in this PR that is disabling max-len. We're looking for a change in code and disabling max-len as a last resort.**

For example: 

dropdown.stories line 97
```
ESLint doesn't like this for max length
        <svg
          class="sprk-c-Icon sprk-c-Icon--filled-current-color sprk-c-Icon--stroke-current-color sprk-u-mls"
          viewBox="0 0 100 100">
          <use xlink:href="#chevron-down" />
        </svg
```

Would turn into:
```
        <svg
          class="
            sprk-c-Icon 
            sprk-c-Icon--filled-current-color 
            sprk-c-Icon--stroke-current-color
            sprk-u-mls
          "
          viewBox="0 0 100 100">
          <use xlink:href="#chevron-down" />
        </svg>
```

##  💡 **Please go through where you disabled and see if you can correct this way. If it can't be fixed, disable max-len is fine, but it should be a last resort.**

Reply:
@RVMendoza Thanks! This makes sense. I was concerned about breaking lines for `class=` because it is used in the code examples but this solves those.